src/Parser/DeathKnight/Frost/Modules/Features/HardHowlingBlastCasts.js

```diff
@@ -10,38 +10,26 @@ class HardHowlingBlastCasts extends Analyzer {
     abilityTracker: AbilityTracker,
   };
 
-  rimeProcs = 0;
-  castsWithRime = 0;
   castsWithoutRime = 0;
-  hardHowlingBlastCasts = 0;
-  nonrimedHB = 0;
 
   on_byPlayer_cast(event) {
     const spellId = event.ability.guid;
     if (spellId !== SPELLS.HOWLING_BLAST.id) {
       return;
     }
-    if (this.selectedCombatant.hasBuff(SPELLS.RIME.id, event.timestamp)) {
-      this.castsWithRime += 1;
-    } else {
+    if (!this.selectedCombatant.hasBuff(SPELLS.RIME.id, event.timestamp)) {
       this.castsWithoutRime += 1;
     }
   }
-  on_byPlayer_applybuff(event) {
-    const spellId = event.ability.guid;
-    if (spellId === SPELLS.RIME.id) {
-      this.rimeProcs += 1;
-    }
-  }
 
   statistic() {
     this.nonrimedHB = this.castsWithoutRime;
     return (
       <StatisticBox
         icon={<SpellIcon id={SPELLS.RIME.id} />}
-        value={this.nonrimedHB}
+        value={this.castsWithoutRime}
         label="Howling Blasts without Rime proc"
-        tooltip="You should aim to get this as close to 0 as possible."
+        tooltip="You should aim to get this as close to 0 as possible.  It is a DPS loss to cast Howling Blast without Rime."
```

Comment:
What about on pull to apply Frost Fever and for AOE?

Reply:
There is almost no situation where it is worth it to cast Howling Blast without Rime.  Rime procs happen often enough there's no need to hardcast it just to apply the dot.  You will get a Rime proc in the first 10 seconds and on AoE it deals half damage to secondary targets.  You would need to be able to hit 10+ targets sustained to consider just doing HB spam.  Though now that you mention it, I should add that its okay to hardcast if you are forced to move off the boss for enough time that you're capping rune (e.g. Intermission phases on KJ you would spam it to generate RP)